docs/reference/TensorFlow.md

```diff
@@ -1,405 +1,370 @@
 
-2.  Deploy the sample to your K66F
+    This creates a folder in `tensorflow/lite/micro/tools/make/gen/mbed_cortex-m4/prj/micro_speech/mbed` that contains the source and header files for the example application, Mbed OS driver files, and a README.
 
+1. Navigate into the `mbed` folder:
 
-3.  Use new trained models to recognize different words
+    ```
+    cd tensorflow/lite/micro/tools/make/gen/mbed_cortex-m4/prj/micro_speech/mbed
+    ```
+1. Connect your board to your computer over USB. On your board, when the Ethernet port is facing you, the micro USB port is to its left.
+    Your board appears as storage on your computer. If your system does not recognize the board with the `mbed detect` command, follow the instructions for setting up [DAPLink](https://armmbed.github.io/DAPLink/?board=FRDM-K66F).
+    
+1. Execute the following to flash the application onto the board:
 
-# Download and build the sample application
+    ```
+    mbed config root .
 
-**Install Arm toolchain and Mbed CLI**
--   Download [Arm cross compilation](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads) toolchain. Select the correct toolchain for the OS that your computer is running. For Windows users, if you have already set up the Linux virtual environment, install the toolchain there.
+    mbed deploy
+    
+    mbed update mbed-os-6.3.0
 
--   To build and deploy the application, we will use the [Mbed CLI](https://github.com/ARMmbed/mbed-cli). We recommend that you install Mbed CLI with our installer. If you need more customization, you can perform a manual install. Although this is not recommended.  
+    mbed compile -m K66F -t GCC_ARM --flash --sterm
+    ```
 
-    If you do not already have Mbed CLI installed, download the installer:  
+    <!--The install page for Mbed CLI now requires 3.7. Why are we asking for 2.7?-->
+    <!--need to explain what `config root .` and `deploy` do and why we need them - neither one is part of a standard workflow where you use Mbed CLI to import an application, so this is a special case-->
+    <!--and why are we compiling here? We compile again two steps down, with the flash parameter-->
 
-    [Mac installer](https://github.com/ARMmbed/mbed-cli-osx-installer/releases/tag/v0.0.10)
--   After Mbed CLI is installed, tell Mbed where to find the Arm embedded toolchain.  
-  ```    
- mbed config -G GCC_ARM_PATH <path_to_your_arm_toolchain>/bin
- ```
-
-**Important:** We recommend running the following commands from inside the Mbed CLI terminal that gets launched with the Mbed CLI Application. This is because it will be much quicker to set up, because it resolves all your environment dependencies automatically.
+1. Speak into the board's microphone: Saying "Yes" will print "Yes" and "No" will print "No" on the serial port:
 
+    ```
+    Heard yes (208) @116448ms  
```

Comment:
What's 208?

Reply:
Below the output (on line 116) we could add:
> the output shows found_command, score and current_time as described in the command_responder.cc file.